Resources/Prototypes/Catalog/Fills/Lockers/heads.yml

```
@@ -22,7 +22,7 @@
       - id: QuartermasterIDCard # Delta-V
       - id: LunchboxCommandFilledRandom # Delta-V Lunchboxes!
         prob: 0.3
-      - id: WeaponEnergyGunMini #LPP
+      - id: WeaponDisabler #LPP
```

yamllint

[warning] too few spaces before comment

(comments)